main.tex

```diff
@@ -92,75 +87,69 @@ \section*{Introduction}
 %
-An estimated 663 million cases of malaria were averted in Africa between 2000 and 2015 due to public health interventions, of which 68\% were prevented by insecticide-treated bed-nets (ITNs) and @@N\% through indoor residual spraying of insecticides (IRS).
+An estimated 663 million cases of malaria were averted in Africa between 2000 and 2015 due to public health interventions, of which 68\% were prevented by insecticide-treated bed-nets (ITNs) and 10\% through indoor residual spraying of insecticides (IRS) \cite{Bhatt2015}.
 %
-However, over this same period, insecticide resistance has become increasingly prevalent in malaria vector populations.
+However, over this same period, insecticide resistance has become increasingly prevalent in malaria vector populations \cite{Hemingway2016}, \cite{Silva2014}.
 %
-Four chemical classes of insecticides -- organophosphates, carbamates, pyrethroids and organochlorines -- are licensed for use in public health, but only pyrethroids are approved by the World Health Organisation (WHO) for use in ITNs.
+Four chemical classes of insecticides – organophosphates, carbamates, pyrethroids and organochlorines – are licensed for use in public health, but only pyrethroids are approved by the World Health Organisation (WHO) for use in ITNs.
 %
 Pyrethroids are also commonly used for IRS and in agriculture, and mosquito populations are under pressure to evolve molecular mechanisms of pyrethroid resistance.
 %
-There is evidence that pyrethroid resistance has a direct impact on the effectiveness of ITNs and IRS, although assessing the impact on disease prevalence is difficult and has been hampered by the fact that pyrethroid resistance is now so pervasive that it is nearly impossible to find fully susceptible mosquito populations to serve as controls.
+There is evidence that pyrethroid resistance has a direct impact on the effectiveness of ITNs and IRS \cite{NGuessanRCorbelVAkogbetoM2007}, \cite{Toe2014}, although assessing the impact on disease prevalence is difficult and has been hampered by the fact that pyrethroid resistance is now so pervasive that it is difficult to find mosquito populations with pyrethroid susceptibility at baseline to serve as controls \cite{WorldHealthOrganization2016}.
 %
-Nevertheless, the position of the WHO remains that insecticide resistance poses a grave threat to the future of malaria control in Africa (@@REF GPIRM).
+Nevertheless, the position of WHO remains that insecticide resistance poses a grave threat to the huge gains made in reducing malaria across Africa \cite{WorldHealthOrganization2012}, \cite{WorldHealthOrganization2016}.
 %
-Improvements are needed in our ability to monitor resistance, and gaps must be filled in our knowledge of the molecular mechanisms of resistance.
+Improvements are needed in our ability to monitor resistance, and gaps must be filled in our knowledge of the molecular mechanisms.
```

Comment:
Nit, but this hangs at the end for me - molecular mechanisms of what? Could we say, "... molecular basis of resistance"? Or something like that?